src/components/InventoryTabs/ConventionalSystems/useTableActions.js

```diff
@@ -0,0 +1,120 @@
+import React, { useCallback } from 'react';
+import {
+  GENERAL_GROUPS_WRITE_PERMISSION,
+  NO_MODIFY_GROUPS_TOOLTIP_MESSAGE,
+  NO_MODIFY_GROUP_TOOLTIP_MESSAGE,
+  NO_MODIFY_HOST_TOOLTIP_MESSAGE,
+  REQUIRED_PERMISSIONS_TO_MODIFY_GROUP,
+  REQUIRED_PERMISSION_TO_MODIFY_HOST_IN_GROUP,
+} from '../../../constants';
+import { ActionDropdownItem } from '../../InventoryTable/ActionWithRBAC';
+
+// some actions are hidden under feature flag
+export default (
+  groupsEnabled,
```

Comment:
We can simplify the hook interface by removing this argument and calling the feature hook inside the useTableActions instead.